lib/src/components/NxRadio/__tests__/NxRadio.test.tsx

```diff
@@ -21,146 +22,131 @@ describe('NxRadio', function() {
     children: undefined
   };
 
-  const getShallow = getShallowComponent<Props>(NxRadio, simpleProps);
+  const renderEl = rtlRenderElement<Props>(NxRadio, minimalProps);
+  const quickRender = rtlRender<Props>(NxRadio, minimalProps);
 
   it('renders a <label> containing a radio <input> and .nx-radio__circle and .nx-radio__outer-circle  elements',
       function() {
-        const shallowRender = getShallow();
+        const el = renderEl()!;
 
-        expect(shallowRender).toMatchSelector('label.nx-radio');
+        expect(el.tagName).toBe('LABEL');
+        expect(el).toHaveClass('nx-radio');
 
-        expect(shallowRender.find('input')).toHaveProp('name', 'color');
-        expect(shallowRender.find('input')).toHaveProp('type', 'radio');
-        expect(shallowRender.find('input')).toHaveProp('disabled', false);
-        expect(shallowRender.find('input')).toHaveProp('checked', false);
+        expect(el.querySelector('input')).toHaveAttribute('name', 'color');
+        expect(el.querySelector('input')).toHaveAttribute('type', 'radio');
```

Comment:
> Probably better to query by role rather than by element. I also don't think there's a need to check the `type` if you query by role.

Like this?: 5cab60ae3b1ac98a479a240989a9c696753669d3

Reply:
> Like this?: https://github.com/sonatype/sonatype-react-shared-components/commit/5cab60ae3b1ac98a479a240989a9c696753669d3

On the lines below that too though

Reply:
> > Like this?: [5cab60a](https://github.com/sonatype/sonatype-react-shared-components/commit/5cab60ae3b1ac98a479a240989a9c696753669d3)
> 
> On the lines below that too though

I thought I got them all: f587d8d25edfbd1c7cac6f46abd13da5f6e45941